.github/workflows/release.yml

```diff
@@ -64,6 +64,17 @@ jobs:
       - name: Checkout Repo
         uses: actions/checkout@v4
 
+      - name: Determine tag from myst-to-react package.json
+        id: get_tag
+        run: |
+          VERSION=$(jq -r '.version' packages/myst-to-react/package.json)
+          if [ -z "$VERSION" ]; then
+            echo "Unable to read version from packages/myst-to-react/package.json" >&2
+            exit 1
+          fi
+          TAG="myst-to-react@$VERSION"
```

Comment:
we are calling these tags / releases `myst-to-react` because we're getting the version from that package. if `myst-theme` is better we could do that? but this seemed simpler